flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/stream/table/ValuesITCase.java

```diff
@@ -332,7 +332,23 @@ public void testRegisteringValuesWithComplexTypes() {
         mapData.put(1, 1);
         mapData.put(2, 2);
 
-        Row row = Row.of(mapData, Row.of(1, 2, 3), new Integer[] {1, 2});
+        Row row = Row.of(mapData, Row.of(1, 2, 3), new int[] {1, 2});
```

Comment:
The type of this is ROW < .., .., ARRAY<INT NOT NULL>>. So we have to provide `new int[] {1, 2}` to pass the `containsExactly`